src/main/scala/com/socrata/snapshotter/SnapshotService.scala

```diff
@@ -16,42 +16,43 @@ import com.socrata.http.client.{Response, SimpleHttpRequest, RequestBuilder}
 import com.socrata.http.server.implicits._
 import com.socrata.http.server.responses._
 import com.socrata.http.server.routing.SimpleResource
+import com.socrata.http.server.util.RequestId
 import com.socrata.http.server.{HttpResponse, HttpRequest, HttpService}
 
 import org.joda.time.{DateTime, DateTimeZone}
 import org.apache.commons.io.IOUtils
 import org.slf4j.LoggerFactory
 
-case class SnapshotService(coreClient: CuratedServiceClient, blobStoreManager: BlobStoreManager, gzipBufferSize: Int) extends SimpleResource {
+case class SnapshotService(sfClient: CuratedServiceClient, blobStoreManager: BlobStoreManager, gzipBufferSize: Int, basenameFor: (ResourceName, DateTime) => String) extends SimpleResource {
```

Comment:
I decided snapshotter should talk ONLY to SF instead of to both SF and core.  So yeah, there can only be one truth.